test/0dt/mzcompose.py

```diff
@@ -98,7 +98,6 @@ def workflow_read_only(c: Composition) -> None:
         CREATE CLUSTER cluster SIZE '2-1';
         GRANT ALL ON CLUSTER cluster TO materialize;
         ALTER SYSTEM SET cluster = cluster;
-        ALTER SYSTEM SET enable_0dt_deployment = true;
```

Comment:
This gets set to true elsewhere, yeah?

Reply:
Yes, it's enabled because we get the default system parameters for 0dt for the entire test via `SYSTEM_PARAMETER_DEFAULTS = get_default_system_parameters(zero_downtime=True)`